core/storage/classifier/gae_models.py

```diff
@@ -181,6 +181,25 @@ def create(
         training_job_instance.put()
         return instance_id
 
+    @classmethod
+    def query_training_jobs(cls, cursor=None):
+        """Generates a unique id for the training job of the form
+        {{exp_id}}.{{random_hash_of_16_chars}}
+
+        Args:
+            cursor: str. It represent the position of cursor in the query
```

Comment:
No, this is the actual cursor, it doesn't represent the position of the cursor. PTAL at other docstrings in the codebase for 'cursor' -- we use this concept elsewhere.